Doxyfile

```diff
@@ -277,7 +277,10 @@ ALIASES                = \
     "es_extension2{3}=<a href=\"http://www.khronos.org/registry/gles/extensions/\1/\3.txt\"><tt>\1_\2</tt></a>" \
     "webgl_extension{2}=<a href=\"https://www.khronos.org/registry/webgl/extensions/\1_\2/\"><tt>\1_\2</tt></a>" \
     "fn_al{1}=`al\1()`" \
-    "def_al{1}=`AL_\1`"
+    "fn_alc{1}=`alc\1()`" \
+    "def_al{1}=`AL_\1`" \
+    "def_alc{1}=`ALC_\1`" \
+    "al_extension{2}=<a href=\"http://icculus.org/alextreg/index.php?operation=op_showext&extname=\1_\2\"><tt>\1_\2</tt></a>"
```

Comment:
Looks like I used my own command incorrectly :D
Here's what I suggest I do: 
 - add alc_extension
 - remove all the eax, since I will not implement them any time soon.